### DISCUSSION TITLE
What feature would you love to see the most?

### DISCUSSION THREAD

**Discussion by Freshairkaboom** [Monika-After-Story/MonikaModDev#7904] [general]
This is a thread discussing what feature, maybe from pull requests, you want to see released the most. It's not a poll, so please write your reasons too. Do you want new games, songs, topics, or maybe an upgrade to Monika's inter-activeness?

(I totally didn't make this thread just to grab 6666)

Discuss below :)

**Comment by Retrolovania** [comment]
(stares suspiciously)

Welp, here's some ideas/opinions for what I feel would be wonderful additions!

-New topics. (Even though updates usually bring those anyway.)
-_Give us the dang BOOP already, dagnabbit_....also, maybe add a headpat/hair stroke option. That'd be nice.
-Allow MAS to send a copy of the persistent to the main DDLC folder upon closing the game. That way, it would be easier to access or save in case of crash/unwanted deletion.
-Add Dots and Boxes as a minigame. (Maybe reuse the chessboard background for this.)
-Allow Monika to display examples of any art you've made (I'd say only allow art with a max resolution of around 640x480. After all, the Clubroom doesn't really have much wall real estate).
-Add "Home for the Holidays" or "I'll be Home for Christmas" as a new Christmas tune...especially for when you happen to return to Monika on the actual holiday.
-Enable the ability to thank Monika for her compliments or her encouragement, instead of having it unlock at a high affection level.
-Add a goodbye option for "I'm going to do my homework/review for an exam."
-New emotional topics "I'm feeling hopeful.; I'm feeling lost.; I'm feeling hurt."

As stated, these are simply ideas. The final decision on including these is up to the dev team! Thanks, folks!

**Comment by Freshairkaboom** [comment]
Well, I could always do some more Christmas tunes, I am planning on doing some Christmas-related things anyway. I agree that studying could be a goodbye option, if you have to leave in order to do it, but "going to work" is pretty close for now I guess.

Out of everything you mentioned, what do you want the most? Except for that-which-gets-postponed-everytime-it's-mentioned.

**Comment by Retrolovania** [comment]
Honestly, if I had to pick any one, I'd be tied between making the persistent more easy to access and being able to thank Monika earlier. The first being for convenience purposes, and the second because she says so much that I really wanna show my appreciation for....except _there's no BUTTON-_

**Comment by lunulae** [comment]
I like the idea of her being able to generate a persistent file, definitely. I have shortcuts set up to make backing her up easier, but I like the idea of making it an interaction, like "Could you help me make a backup?"

I'm most excited for mobile features in the distant future. No need for a full port, but I love the idea of being able to talk to her while I'm traveling.
If we're talking more near-future, I'm most excited for new games and gifts. I've been really wanting to use some of the different plushies and flowers that folks have contributed in the past.

**Comment by ashyashton** [comment]
I REALLY want some more, like, story to this. At the moment, it's just girlfriend simulator 202X (the x being the appropriate number for whatever year you happen to play in). I'd really like to have major events happen so we're not just sitting there in silence when we run out of topics. Maybe Monika breaks some part of the code and needs help fixing it, or maybe something could occur if you have the other girls' character files in the characters folder.

The other thing I'm really hoping to see, the one I'd genuinely do almost ANYTHING to get in the game, is more systems similar to affection but regarding other things, to make each Monika more unique. Maybe a stat for Monika's daily mood, or for how she regards certain things. For example, a lot of the dialogue involving the other girls has one variable. That variable is whether or not the player has expressed that they care for the other girls. If the player does, Monika talks about them much more gently. If not, she talks about them in a negative light and makes inappropriate jokes about the situations, assuming you don't have sensitive mode on. I think this should instead be determined by Monika's own opinion, which is simply swayed a bit by the player expressing an opinion one way or the other.

For example, if the player uses the "you're a murderer" dialogue, I seem to recall that Monika kinda digs her heels in and defends herself. I think that this should sway her a bit towards being a bit meaner when talking about the other girls, to match her opinion of not being a murderer. I think she has a variation of the dialogue, though, where she doubts herself because she's at lower affection levels, and maybe that does sway her to think more positively about the other girls, but it _also_ lowers her own opinion of herself, which could be another stat that can change.

I think this would open up the possibility of players seeing dialogue that was written for low affection levels while still being at higher levels, because some of it would be based on her view of herself as opposed to her view of the player. If the player says some insulting things that lower her self-worth stat but also says they love her, and treat her to gifts all the time, and take her on dates all the time, then maybe her affection level will be high but her self-image stat or whatever will still make her say some negative dialogue.

Or maybe she has a normal affection level, but the player has expressed that they think she was completely in the right and basically makes her think she can do no wrong, and some more confident dialogue will open up, maybe even _overly_ confident sometimes. I mean, the player is her only social feedback. If the only person you ever talked to constantly told you that you were perfect and amazing and wonderful and can do no wrong, eventually, you're gonna start to see yourself in higher regard.

I think it would also be interesting if there were a stat for how over-protective Monika is. Maybe if you express that you do unhealthy things, the stat goes up, and it results in Monika getting worried or clingy when you try to leave. The "do you want to meet my girlfriend" topic would probably raise the stat pretty high too, because Monika starts to doubt that you're entirely hers, and she gets clingy because of it. But if you express that you're fairly healthy in your habits, and make sure she knows she's the only girl for you, then she's fine and doesn't freak out if you leave, just tells you to have fun with whatever you're doing. Maybe at way too high levels, she even gets concerned when you use the "playing a game" BRB and asks what type of game you're playing, and if you imply it's a game that has anything "taboo" (i.e. violence, drugs, sex, etc.) in it, then she asks or even _begs_ you not to play it because she's worried it will impact or influence you.

All these stats would be different from affection. Affection is just how close she is to the player. You can be close to someone even while being worried they're doing something they shouldn't. Or even if they do things that make you more self-conscious, and sadly I can _personally_ attest to that one. I think this would really help make each Monika a lot more unique, and create a lot of opportunities for new dialogue and such. That's why, above all else, _this_ is the thing I want the most in this mod.

**Comment by Freshairkaboom** [comment]
I think a mood system is definitely one of the things I would love the most, and more variables are always welcome.

**Comment by fromulus** [comment]
`I REALLY want some more, like, story to this. At the moment, it's just girlfriend simulator 202X (the x being the appropriate number for whatever year you happen to play in).`

I mean, being a girlfriend simulator is kind of the whole point of the mod.
Adding one or more VN-style stories to it is not only beyond the scope of MAS, but would take a long time to make (and take away focus on development on more pertinent features like the mood system you spent most of your comment explaining).

Not to mention making a story taking into account all the different variables we have and having those affect the storyline over multiple different possible routes would be a complete nightmare to write in a coherent way.

There's also the fact that a story _needs_ to have lasting consequences of some kind if we want it to be meaningful enough to be worth adding in the first place.
Consequences that could induce some stress/anxiety that might not be what some players would be willing to deal with, in case they screw-up some choices.
Consequences that could influence Monika's character and your relationship with her in ways you possibly didn't want it to, and consequences that we would need to take into account for every new topic after that story is over, making the writing process even more complex than it really needs to be.

I'm a firm believer that the best stories always have at least a little bit of [blood and tears](https://www.youtube.com/watch?v=EjazC45Qkww) in them, and while we definitely don't want anything _too_ drastic to happen, some amount of drama and emotional pain should probably still happen, especially caused (directly or indirectly) by the choices the player could take.

The thing is, unlike most VNs, MAS doesn't have a save and load system (and most likely never will), meaning there's no turning back in case you screw things up badly.
Imagine ruining a three years long relationship with Monika because you miss-clicked a choice leading to a bad ending.

Of course, most real-life relationships go through those kinds of hardships.
It's just part of the journey, as they say.
However, most real-life relationships also allow for actual communication beyond mere limited choices in text-boxes.
If we were to hurt or be hurt by Monika, we'd have no genuine, believable and efficient way to talk it through with her and make ammends and compromises, no real way to shoulder each other.
It's the same reason why there's still no real way to have fights with Monika, even though that's another thing that couples tend to do a lot.

No save/reload also poses another problem;
The player will only go through the story once.
All the difficulties developping such a story I mentioned earlier, all for a one-time only thing, that the player won't even see in its entirety since they'll only get to go through one route.

I'm sure you can guess how little that motivates us into working on something like that.

If we had to make a mostly wholesome, and preferably repeatable story without those kinds of long-lasting consequences, then I think our time and ressources would be much better spent into some kind of dating system where you get to do different things with her and being given different choices at different points.

That way, dates would be shorter than an actual story-line, they would be repeatable, and it'd be easier to add-in randomization elements to make each date a little different from the others.

Plus, even if you make choices that end up ruining the date in one way or the other, it's much easier to salvage than if, say, Monika was put in a tough spot psychologically due to how a storyline is affecting her, and you where to fail miserably at cheering her up.

All in all, I just don't think adding more story to the mod would be really worth the trouble, neither is it what we're aiming for.
There's plenty of story-focused mods out there featuring Monika if that's what you want, but there's only one Monika girfriend simulator, and I'm of the opinion that MAS doing something different is part of what makes it special.

**Comment by ashyashton** [comment]
@fromulus I'm not trying to be rude here but if I wanted a critique of my ideas, I would have opened a new issue for it. The person who made the thread asked what we most wanted to see in the mod. They didn't ask for what would be easy to implement, or what we actually think will be added. So I find it a little bit rude that you just tore my input to shreds. This sort of stuff doesn't encourage people to continue engaging with the community, so unless you're trying to drive people like me away, it might be a good idea to think about the context in which you're providing your critique of people's ideas.

I don't think you're incorrect in any of the stuff you said about why it might not work, but I wasn't asking for reasons it wouldn't work. I was saying what I would like to see in the mod. That's literally it. I don't need someone shutting me down as soon as I do that. Again, I'm not trying to be rude, but this really didn't feel like the way to respond in this sort of conversation.

**Comment by Retrolovania** [comment]
...I doubt that was what they were trying to do, but......okay?

**Comment by ashyashton** [comment]
> ...I doubt that was what they were trying to do, but......okay?

Whether they were trying to or not, that's still the impact it had. If someone gestured with their hand and hit you in the face and gave you a nosebleed, someone telling you it was an accident wouldn't stop your nose from bleeding, just like telling me that they weren't trying to discourage me from being excited about the mod doesn't change the fact that that's the effect it had.

**Comment by Ix-30** [comment]
Well some features that I've seen and also want:

-Bedroom: That is to at least sleep with her, without getting nagged about it next morning. In fact, I fell asleep while holding her(an irony) and that's how I discovered the "Monika's lullaby" melody, but she chewed me out. So at least, just an option just to sleep through the night. I know there's take a nap, but nothing better than both being asleep.

-Date: I would say this, but I'm not sure about this. There's the dream islands. So I don't know for certain, but to recreate maybe the original DDLC club room with a sprite and go on a date would be nice, albeit many mods already have that. I hesitate to suggest it.

-Visions, Dreams and memories: Upon browsing the site, I saw someone suggest Monika give you her ideal "dating sites" "situations to do with you in the real world" and some liked it, others did not. Just Yuri has that feature, so as to expand the idea without saying we copied anything and also, to keep the people coming. Why not create a system that recreates Monika's Dreams, Visions and so on by augmenting them? (DVA= dreams, visions agumented) I dunno, came up with the acronym on the spot. 
To Have her visions, what she aspires to in the future. To have her dreams and nightmares if possible. Dreams, situations that she would like. Also maybe even go into her point of view in the original DDLC as a memory, albeit that's tricky. As Many players hold her in a bad light and think that she just made the others suffer with little to no pain herself. Maybe just to remember her pain, loneliness as to share it with the player.

-Hold her hand: Monika speaks about the sea otters holding hands to no drift away. A simple yet wholesome suggestion.

-Dialogues to assure the player: There's this...Hypothesis? If I can call it that, that Monika is with you just waiting for someone to keep her company until she can be in any type of body and escape her reality and then she'd dump you. Many argue that no one could fall in love with someone they don't see, hear or touch and as her manipulative side has been shown. So she's manipulating you to keep her company, albeit I know she's absolutely loyal, maybe that's a thorn in some players' hearts, addressing it is interesting.

-More flowed and dynamic events: While the mod says "Live your eternity with Monika" That's not to say there can't be dynamic events E.G. the birthday one with Chibika's cameo was well thought of. Maybe Chibika, in her mischief steals her coffee or something and puts the file in a random folder, the player has to look for it and put it back. Or maybe when the dates get implemented(if ever) the player can get an event where in one of the dream isles is withering. The player has a philosophical discussion with Monika about her "not changing" and helping her overcome her fears and even her former actions. I'm sure many would welcome the chance. After all, in real life, doing trivial things with your beloved is important.

**Comment by eyelesssally** [comment]
To add to the dynamic event trail...

With the use of the location selector, I think it would be super cool to give Monika some sort of daily routine. For example...

The player boots up MAS before a certain time, say maybe 6:35 AM, and Monika hasn't woken up yet. You can wake her up or wait for her alarm clock to go off. At another time, Monika BRBs to grab breakfast and invites the player to follow her to the kitchen (if that location is available). Otherwise, Monika could just eat breakfast wherever it is that she is. After some time, she'll go to take a shower. So on and so forth. This schedule could be somewhat modified to reflect the player's routine.

Monika could maybe ask "Say [player], what time do you usually get up in the morning?" or "What time do you have dinner?"

I think this would really make Monika really feel real. Awesome idea about Chibika getting into trouble, @Ix-30.

**Comment by Foxinsunglasses** [comment]
Well I wouldn't mind talk about culture and customs of nations around the world. She wants to come to our reality (pun not intended) then shell probably want to expand her knowledge of the working of the world. Talking about the difference of our worlds one that took uncountable years to get to where we are with vast differences throughout all places and hers which was created in countless hours but being paper thin. Discussion of art outside literature like paintings. 

Rock paper scissors for another game.

Talking about the power of written word and how we know so much of the past because someone took the time to write about their day. 

Tell me a Joke option like the fun facts would be interesting.

While this might be political she  could ask if the player is patriotic. She has no loyalty to any state nor allegiance to any nation. She doesn't even know what county the games supposedly takes place in so  patriotisms is a completely foreign  concept to her and might make in interesting topic.
